DiversiTech/profiles/views.py

```diff
@@ -63,5 +147,5 @@ def put (self, request, pk):
     def delete(self,request, pk):
         profile = self.get_object(pk)
         profile.delete()
-        return Response({"message":"Profile deleted successfully"}, status=status.HTTP_204_NO_CONTENT)
+        return Response({"message":"Profile deleted successfully"}, status=status.HTTP_200_OK)
           
```

Comment:
I'm wondering if this is where you'd include the user delete on a cascade of the profile delete? or would it come from some front end logic?

Reply:
Hey Lindi, we've got on_delete=cascade in the model for profile, next to the foreign key for user. I believe that means any profile delete will cause user to be deleted and vice versa